src/tests/sampleData/sampleReviews.json

```diff
@@ -3,32 +3,72 @@
     {
       "id": 1,
       "title": "Sample Proposal 1 - Dual proposal",
-      "criterion": [],
-      "users": ["admin", "user"]
+      "criterion": [
+        {"title": "Criterion1", "shortTitle": "Crit1", "text": "Criterion 1 text for display to user for guidance.", "weight": 50, "panel": false},
+        {"title": "Criterion2", "shortTitle": "Crit2", "text": "Criterion 2 text for display to user for guidance.", "weight": 10, "panel": false},
+        {"title": "Criterion3", "shortTitle": "Crit3", "text": "Criterion 3 text for display to user for guidance.", "weight": 10, "panel": false},
+        {"title": "Criterion4", "shortTitle": "Crit4", "text": "Criterion 4 text for display to user for guidance.", "weight": 20, "panel": false},
+        {"title": "Criterion5", "shortTitle": "Crit5", "text": "Criterion 5 text for display to user for guidance.", "weight": 10, "panel": true}
+      ],
+      "reviewers": "peer",
+      "users": ["admin", "user"],
+      "panelReview": "guest"
     },
     {
       "id": 2,
       "title": "Sample Proposal 2 - Admin only",
-      "criterion": [],
-      "users": ["admin"]
+      "criterion": [
+        {"title": "Criterion1", "shortTitle": "Crit1", "text": "Criterion 1 text for display to user for guidance.", "weight": 50, "panel": false},
+        {"title": "Criterion2", "shortTitle": "Crit2", "text": "Criterion 2 text for display to user for guidance.", "weight": 50, "panel": false},
+        {"title": "Criterion3", "shortTitle": "Crit3", "text": "Criterion 3 text for display to user for guidance.", "weight": 50, "panel": false},
+        {"title": "Criterion4", "shortTitle": "Crit4", "text": "Criterion 4 text for display to user for guidance.", "weight": 50, "panel": false},
+        {"title": "Criterion5", "shortTitle": "Crit5", "text": "Criterion 5 text for display to user for guidance.", "weight": 50, "panel": true}
+      ],
+      "reviewers": "peer",
+      "users": ["admin"],
+      "panelReview": "guest"
```

Comment:
Ditto to my question about stories instances like these as variables somewhere...

Reply:
Yes, all of this sample data is just placeholder in my attempt to mimic how the DB might be storing items on the back end. This is going to be updated.